### PR TITLE
Divide into Describe and Price

### PR DESCRIPTION
**Pull request by kiroj305**
Divide the Nft Edit scenario into 2 steps

### CODE REVIEW

File: lib/utils/constants.dart [hunk 1]
```diff
@@ -169,6 +170,7 @@ const kHintPrice = '10.87';
 const kHintHashtag = 'Type in';
 const kHashtagsText = 'Hashtags (optional)';
 const kAddText = 'Add';
+const kNetworkFeeWarnText = 'A network fee of 10% the listed price is required for all transactions that occur on-chain';
```
Comment (mustafapylons) [inline lib/utils/constants.dart]: need to add another "of" after the word "10%"

**Review by mustafapylons — COMMENTED** [review]
One small change, otherwise code is good.

File: lib/utils/constants.dart (hunk 1 above)
Comment (kiroj305) [reply lib/utils/constants.dart]: @mustafapylons , it's fixed now.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Comment by kiroj305** [comment]
@kjawadDeveloper2 , please take a review when you have time. Your #71 was merged to this.

**Comment by kjawadDeveloper2** [comment]
Your code has no issue. I reviewed your ui on android device working fine . Write now checking on tablet

**Comment by kjawadDeveloper2** [comment]
img
1) So your Y is cutting from the bottom.
2) The whole Add button is not tappable its only tappable in the middle where it says Add.

**Pull request merged**